cmsis/arm_math.h

```diff
@@ -345,8 +345,8 @@ extern "C"
 #define DELTA_Q31          (0x100)
 #define DELTA_Q15          0x5
 #define INDEX_MASK         0x0000003F
-#ifndef PI
```

Comment:
Nah, it won't. It's for testing really, I have a change for mcr20a-rf-driver that renames the register, rather than the PI.